File: scenarios/04_transitive_via_export/child/src/test/java/transitive_via_export/parent/ChildTest.java
package transitive_via_export.parent;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

class ChildTest {
  @Test
  public void print_hello_three_times() {
    assertTrue(new Child().canIUseGuava());
  }
}

File: scenarios/12_kotlin/child/test/kotlin_example/child/ChildTest.kt
package kotlin_example.child

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ChildTest {
  @Test
  fun `say hello`() {
    assertEquals("hello", Child().sayHello())
  }

  @Test
  fun `say hello and la 4 times`() {
    assertEquals("hellolalalala", Child().sayHelloAndLa())
  }
}